spec/01-unit/020-sandbox_spec.lua

```diff
@@ -240,6 +240,12 @@ describe("sandbox functions wrapper", function()
           _G.kong.configuration = deep_copy(base_conf)
         end)
 
+        it("has access to string.rep", function()
+          for _, m in ipairs(modules) do
+            assert.same("aaa", sandbox.sandbox("return string.rep('a', 3)")())
```

Comment:
`m` is not used.  is that loop needed?

an `('a'):rep(3)=='aaa'` test would be good too

Reply:
Done!